apps/etterlatte-vilkaarsvurdering/src/main/kotlin/vilkaarsvurdering/VilkaarsvurderingRoutes.kt

```diff
@@ -35,6 +35,12 @@ fun Route.vilkaarsvurdering(vilkaarsvurderingService: VilkaarsvurderingService)
                 } catch (e: VirkningstidspunktIkkeSattException) {
                     logger.info("Virkningstidspunkt er ikke satt for behandling $behandlingId")
                     call.respond(HttpStatusCode.PreconditionFailed)
+                } catch (e: BehandlingstilstandException) {
+                    logger.info(
```

Comment:
logger.error på alle disse i steden for info?

Reply:
tenker at det er en feil som skjer når bruker klikker runt i GUI men er i feil tilstand, så litt kjipt å logge error for det. er liksom et "forventet feil" eller hva man sier.